content/post/kubedb-v2022.12.28/index.md

```diff
@@ -0,0 +1,389 @@
+
+**Supported version:** Support for Kafka version 3.3.0 is added in this release.
+
+Following is a sample YAML to provision TLS secured kafka cluster with 2 dedicated controller nodes and 3 dedicated broker nodes of version `3.3.0` in demo namespace. Create an Issuer named `kafka-ca-issuer` using cert-manager prior to applying this YAML.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: Kafka
+metadata:
+  name: kafka-sample
+  namespace: demo
+spec:
+  version: 3.3.0
+  enableSSL : true
+  tls:
+    issuerRef:
+      apiGroup: cert-manager.io
+      name: kafka-ca-issuer
+      kind: Issuer
+  storageType: Durable
+  terminationPolicy: Delete
+  topology:
+    broker:
+      replicas: 3
+      storage:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+        storageClassName: standard
+    controller:
+      replicas: 2
+      storage:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+        storageClassName: standard
+```
+
+## ProxySQL
+**MariaDB and Percona-XtraDB backend:**
+
+From this release we have added support for `MariaDB` and `Percona-XtraDB` as ProxySQL backend. Both KubeDB managed and external databases are allowed to be set as backend.
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: ProxySQL
+metadata:
+  name: proxy-server
+  namespace: demo
+spec:
+  version: "2.4.4-debian"
+  replicas: 3
+  mode: Galera
+  backend:
+    name: sample-mariadb
+  syncUsers: true
+  terminationPolicy: WipeOut
+  healthChecker:
+    failureThreshold: 3
+```
+
+**Vertical Scaling Ops-request :**
+
+We have added vertical scaling ops-request for ProxySQL in this release.
+
+```yaml
+apiVersion: ops.kubedb.com/v1alpha1
+kind: ProxySQLOpsRequest
+metadata:
+  name: vertical-scale
+  namespace: demo
+spec:
+  type: VerticalScaling  
+  proxyRef:
+    name: proxy-server
+  verticalScaling:
+    proxysql:
+      requests:
+        memory: "1200Mi"
+        cpu: "0"
+      limits:
+        memory: "1200Mi"
+        cpu: "0"
+```
+
+**Autoscaler:**
+Like other KubeDB managed databases from now you will have autoscaling support for ProxySQL.
+
+We support compute (to autoscale CPU & memory resources) autoscaling currently. The structure of those autoscaler yamls is similar for all databases. Compute utilizes the VerticalScale OpsRequest internally while autoscaling. There are two more common fields in the CRDs spec :
+
+spec.proxysql to refer to the actual proxysql server.
+spec.opsRequestOptions to control the behavior of the ops-manager operator. It has two fields: apply and timeout.
```

Comment:
all the yaml fields should be inside backtick mark , like this : `spec.proxysql` to differentiate them from general words/sentences.